### PR TITLE
GROK-13568: Help | Compute: Sensitivity Analysis

### PR DESCRIPTION
**Pull request by vmakarichev**
"Sensitivity Analysis" is added to the Section "Design of experiment" of Compute

### CODE REVIEW

**Review by lbankurova — APPROVED** [review]

File: help/compute/compute.md [hunk 1]
```diff
@@ -443,12 +443,33 @@ Automatic outliers detection Manual outliers markup and annotation Used as an in
 
 ### Sensitivity analysis
 
-* Sample inputs:
-  * by specified number of samples
-  * by a specified distribution or within a range
-  * for a specified set of scalar inputs and/or columns of the matrix input
-* Produce variability analysis for outputs based on the sampled inputs
-* Visualize the results of analysis with Datagrok viewers
+Study what happens to a function when varying its inputs. Datagrok provides investigation of how uncertainty in its output can be divided to different sources of uncertainty in the inputs.
```
Comment (skalkin) [inline help/compute/compute.md:446]: Simplify the message. Something like that (wrote it in a minute, probably can do better)

"Sensitivity Analysis runs the computation multiple times with varying inputs, and analyzes how results are dependent on parameters.

File: help/compute/compute.md [hunk 2]
```diff
@@ -443,12 +443,33 @@ Automatic outliers detection Manual outliers markup and annotation Used as an in
...
+
+To run analysis, click on the icon "Run sensitivity analysis", choose a method, specify inputs & outputs, and press "RUN".
+
+Available types of analysis:
+
+* Grid
```
Comment (skalkin) [inline help/compute/compute.md:452]: I would simplify and put the descriptions inline. Something like that:

* **Grid**: varying parameters in a grid with the specified step.
* **Sobol**: (link): decomposes output variance into fractions, which can be attributed to inputs
...

Comment (skalkin) [reply help/compute/compute.md:452]: Note the use of the bolder font; attract attention to the most important pieces

File: help/compute/compute.md [hunk 3]
```diff
@@ -443,12 +443,33 @@ Automatic outliers detection Manual outliers markup and annotation Used as an in
...
+* Monte Carlo
+* Sobol method
+
+In Grid analysis, inputs are varied within a grid of the specified ranges. When applying [the Monte Carlo method](https://en.wikipedia.org/wiki/Monte_Carlo_method), the studied function is evaluated at randomly taken points.
+
+![monte-carlo-analysis.png](monte-carlo-analysis.png)
+
+[The Sobol method](https://en.wikipedia.org/wiki/Variance-based_sensitivity_analysis) provides a decomposition of the output variance into fractions, which can be attributed to inputs.
```
Comment (skalkin) [inline help/compute/compute.md:460]: Keep simplifying and cutting until there is nothing to cut :) Provides a decomposition => decomposes

File: help/compute/compute.md [hunk 4]
```diff
@@ -443,12 +443,33 @@ Automatic outliers detection Manual outliers markup and annotation Used as an in
...
+
+![sobol-analysis.png](sobol-analysis.png)
+
+First-order Sobol indices indicate the main effect of varying each input alone. Total-order indices measure the contribution to the output variance of each input, including variance caused by its interactions with any other inputs.
+
+When exploring complex models, some evaluations may be of particular interest. In this case, specify inputs set and find the required outputs in the context panel
```
Comment (skalkin) [inline help/compute/compute.md:466]: Probably an animated gif would be great here to show how it's done (and a "wow" factor, too!)

**Review by skalkin — CHANGES_REQUESTED** [review]

File: help/compute/compute.md (hunk 1 above)
Comment (vmakarichev) [reply help/compute/compute.md:446]: Done

File: help/compute/compute.md (hunk 2 above)
Comment (vmakarichev) [reply help/compute/compute.md:452]: Done

File: help/compute/compute.md (hunk 3 above)
Comment (vmakarichev) [reply help/compute/compute.md:460]: Done

**Review by skalkin — APPROVED** [review]
Looks good to me!

**Pull request merged**